python/ribasim/ribasim/model.py

```diff
@@ -456,92 +221,27 @@ def plot(self, ax=None, indicate_subnetworks: bool = True) -> Any:
-        node_attrs, node_instances = zip(*self.nodes().items())
-        node_clss = [node_cls.get_input_type() for node_cls in node_instances]
-        truth_dict = {"T": ">", "F": "<"}
-        assert self.network.node.df is not None
-        assert self.network.edge.df is not None
-
-        if self.discrete_control.condition.df is None:
-            raise ValueError("This model has no control input.")
-
-        for index, row in df_control.iterrows():
-            datetime = row["time"]
-            control_node_id = row["control_node_id"]
-            truth_state = row["truth_state"]
-            control_state = row["control_state"]
-            enumeration = f"{index}. "
-
-            out = f"{enumeration}At {datetime} the control node with ID {control_node_id} reached truth state {truth_state}:\n"
-
-            if self.discrete_control.condition.df is None:
-                return
-
-            conditions = self.discrete_control.condition.df[
-                self.discrete_control.condition.df.node_id == control_node_id
-            ]
-
-            for truth_value, (index, condition) in zip(
-                truth_state, conditions.iterrows()
-            ):
-                var = condition["variable"]
-                listen_feature_id = condition["listen_feature_id"]
-                listen_node_type = self.network.node.df.loc[
-                    listen_feature_id, "node_type"
-                ]
-                symbol = truth_dict[truth_value]
-                greater_than = condition["greater_than"]
-                feature_type = "edge" if var == "flow" else "node"
-
-                out += f"\tFor {feature_type} ID {listen_feature_id} ({listen_node_type}): {var} {symbol} {greater_than}\n"
-
-            padding = len(enumeration) * " "
-            out += f'\n{padding}This yielded control state "{control_state}":\n'
-
-            affect_node_ids = self.network.edge.df[
-                self.network.edge.df.from_node_id == control_node_id
-            ].to_node_id
-
-            for affect_node_id in affect_node_ids:
-                affect_node_type = self.network.node.df.loc[affect_node_id, "node_type"]
-                nodeattr = node_attrs[node_clss.index(affect_node_type)]
-
-                out += f"\tFor node ID {affect_node_id} ({affect_node_type}): "
-
-                static = getattr(self, nodeattr).static.df
-                row = static[
-                    (static.node_id == affect_node_id)
-                    & (static.control_state == control_state)
-                ].iloc[0]
-
-                names_and_values = []
-                for var in static.columns:
-                    if var not in ["node_id", "control_state"]:
-                        value = row[var]
-                        if value is not None:
-                            names_and_values.append(f"{var} = {value}")
-
-                out += ", ".join(names_and_values) + "\n"
-
-            print(out)
+        raise NotImplementedError()
```

Comment:
(As the person who added it) I wonder whether this is worth maintaining

Reply:
Yay to deleting code